client/blocks/get-apps/desktop-download-card.jsx

```diff
@@ -53,32 +61,77 @@ class DesktopDownloadCard extends Component {
 		}
 	}
 
+	getPlatformImage( platform ) {
+		switch ( platform ) {
+			case 'MacIntel':
+			case MAC_LINK:
+				return <SVGIcon aria-hidden="true" name="apple-logo" size="24" icon={ Apple } />;
+			case 'Linux i686':
+			case 'Linux i686 on x86_64':
+			case LINUX_TAR_LINK:
+			case LINUX_DEB_LINK:
+				return <SVGIcon aria-hidden="true" name="linux-logo" size="24" icon={ Linux } />;
+			case WINDOWS_LINK:
+			default:
+				return <SVGIcon aria-hidden="true" name="windows-logo" size="24" icon={ Windows } />;
+		}
+	}
+
 	getTranslateComponents( platform ) {
 		switch ( platform ) {
 			case 'MacIntel':
-				return {
-					firstAvailableLink: this.getLinkAnchorTag( WINDOWS_LINK ),
-					secondAvailableLink: this.getLinkAnchorTag( LINUX_TAR_LINK ),
-					thirdAvailableLink: this.getLinkAnchorTag( LINUX_DEB_LINK ),
-				};
+				return displayJetpackAppBranding
+					? {
+							firstAvailableLink: this.getLinkAnchorTag( WINDOWS_LINK ),
+							secondAvailableLink: this.getLinkAnchorTag( LINUX_TAR_LINK ),
+							thirdAvailableLink: this.getLinkAnchorTag( LINUX_DEB_LINK ),
+
+							firstAvailableIcon: this.getPlatformImage( WINDOWS_LINK ),
+							secondAvailableIcon: this.getPlatformImage( LINUX_TAR_LINK ),
+							thirdAvailableIcon: this.getPlatformImage( LINUX_DEB_LINK ),
+					  }
+					: {
+							firstAvailableLink: this.getLinkAnchorTag( WINDOWS_LINK ),
+							secondAvailableLink: this.getLinkAnchorTag( LINUX_TAR_LINK ),
+							thirdAvailableLink: this.getLinkAnchorTag( LINUX_DEB_LINK ),
+					  };
```

Comment:
@SiobhyB after reviewing [your latest change](https://github.com/Automattic/wp-calypso/pull/69674#discussion_r1012058709) in https://github.com/Automattic/wp-calypso/commit/59edabb39eac9a83792917b4e052316b30742863, I realize I completely misinterpreted the original code. I thought the same keys were being replaced. I realize now that we are conditionally adding keys. I think your original implementation makes total sense. Please feel free to revert https://github.com/Automattic/wp-calypso/commit/59edabb39eac9a83792917b4e052316b30742863 if you'd like. Sorry for the misguidance.